README.md

```diff
@@ -46,7 +46,7 @@
 * 2.2k Ohm Resistor      : amzn.com/B0185FJBJC
 * 3.3v Voltage Regulator : amzn.com/B0758ZTS61
 * TIP102 NPN Transistor  : amzn.com/B00M1GGKIC
-* 940nm LED Array        : amzn.com/B00M1GGKICB07D5FJQ62
+* 940nm LED Array        : amzn.com/B07D5FJQ62
```

Comment:
As per the talk this should say 904nm for the IR LEDs

Reply:
I noticed the same thing and created an issue for it. It looks like 904/905 is the correct value from my quick research. The downside is they are more expensive and harder to find than 940.

Reply:
Does this mean it will not work for just the lidar jammer, or will not work for anything?